### PR TITLE
sage scripts in `kokab` namespace

### PR DESCRIPTION
**Pull request by Qazalbash**
## Summary by CodeRabbit

- **New Features**
	- Simplified labeling in issue templates for feature requests, bug reports, and model requests, enhancing clarity.
	- Introduced a command-line interface in `sage.py` for probabilistic modeling, improving user interaction with astrophysical data analysis.
	- Added utility functions for argument expansion and JSON processing in the `common.py` files, enhancing modularity and usability.
	- Implemented a command-line argument parser in `sage_parser.py` for structured user input in Sage scripts.
	- Added a badge for code coverage in the `README.md`, promoting transparency regarding testing coverage.

- **Bug Fixes**
	- Adjusted data handling in multiple scripts by switching from Polars to Pandas, optimizing compatibility and performance.

- **Documentation**
	- Improved import organization in `__init__.py` for better readability.

- **Refactor**
	- Removed redundant functions in `genie.py`, promoting cleaner code architecture.
	- Enhanced CI workflow with coverage reporting integration in `.github/workflows/ci.yml`.
	- Updated license URL in `.codecov.yml` to use HTTPS for improved security.

### CODE REVIEW

**Comment by Qazalbash** [comment]
@coderabbitai full review

File: pyproject.toml [hunk 1]
```diff
@@ -88,20 +88,22 @@ Issues = "https://github.com/gwkokab/gwkokab/issues"
...
-gwk_joint_plot = "gwkokab._src.scripts.joint_plot:main"
-gwk_scatter2d = "gwkokab._src.scripts.scatter2d:main"
-gwk_scatter3d = "gwkokab._src.scripts.scatter3d:main"
+gwk_batch_scatter2d = "gwkokab.src.gwkokab_scripts.batch_scatter2d:main"
+gwk_batch_scatter3d = "gwkokab.src.gwkokab_scripts.batch_scatter3d:main"
+gwk_chain_plot = "gwkokab.src.gwkokab_scripts.chain_plot:main"
+gwk_corner_plot = "gwkokab.src.gwkokab_scripts.corner_plot:main"
+gwk_joint_plot = "gwkokab.src.gwkokab_scripts.joint_plot:main"
+gwk_scatter2d = "gwkokab.src.gwkokab_scripts.scatter2d:main"
+gwk_scatter3d = "gwkokab.src.gwkokab_scripts.scatter3d:main"
```
[the remark this answers is not in the document]
Comment (Qazalbash) [reply pyproject.toml:97]: I made a mistake `gwkokab.src.` is extra and shouldn't be there. Can you make a commitable suggestion?

[the remark this answers is not in the document]
Comment (Qazalbash) [reply pyproject.toml:97]: Actually, I wanted to remove the prefix `gwkokab.src.` from the path.

**Comment by zeeshan5885** [comment]
I am using the main branch of GWKokab and the cl_structure branch of gwk_analysis.
- I can generate data.
- unable to make plots due to the following error.
`Traceback (most recent call last):
  File "/home/muhammad.zeeshan/.conda/envs/kvenv/bin/gwk_batch_scatter2d", line 5, in <module>
Traceback (most recent call last):
  File "/home/muhammad.zeeshan/.conda/envs/kvenv/bin/gwk_scatter2d", line 5, in <module>
Traceback (most recent call last):
  File "/home/muhammad.zeeshan/.conda/envs/kvenv/bin/gwk_scatter2d", line 5, in <module>
    from gwkokab.src.gwkokab_scripts.batch_scatter2d import main
    from gwkokab.src.gwkokab_scripts.scatter2d import main
    from gwkokab.src.gwkokab_scripts.scatter2d import main
ModuleNotFoundError: No module named 'gwkokab.src'
ModuleNotFoundError: No module named 'gwkokab.src'
ModuleNotFoundError: No module named 'gwkokab.src'
Traceback (most recent call last):
  File "/home/muhammad.zeeshan/.conda/envs/kvenv/bin/gwk_batch_scatter2d", line 5, in <module>
    from gwkokab.src.gwkokab_scripts.batch_scatter2d import main
ModuleNotFoundError: No module named 'gwkokab.src'`

**Comment by zeeshan5885** [comment]
@Qazalbash  Error while making Inference.
`GWKokab exp_rate_integral: 1.9095445523771644
GWKokab exp_rate_integral: 0.504405743653253
GWKokab exp_rate_integral: 0.5903127797182224
GWKokab exp_rate_integral: 0.20473500253870314
GWKokab exp_rate_integral: 0.9468126223937847
GWKokab exp_rate_integral: 0.8578917657005772
GWKokab exp_rate_integral: 0.20193038946949246
GWKokab exp_rate_integral: 0.15821597616995783
GWKokab exp_rate_integral: 0.13531745992925026
GWKokab exp_rate_integral: 1.1996101935130312
GWKokab exp_rate_integral: 0.11266746492240205
GWKokab model_log_likelihood: -907.7316736756782
GWKokab model_log_likelihood: -873.6975188767707
GWKokab model_log_likelihood: -1428.3703076670936
GWKokab model_log_likelihood: -2733.335121601631
GWKokab model_log_likelihood: -1085.7322276476582
GWKokab model_log_likelihood: -1678.1575449748968
GWKokab model_log_likelihood: -1379.8798311346754
GWKokab model_log_likelihood: -2959.1995034199617
GWKokab model_log_likelihood: -798.6979241511883
GWKokab model_log_likelihood: -2393.209749691106
GWKokab log_likelihood: -1983.4145798857467
GWKokab log_likelihood: -1428.7226480255254
GWKokab log_likelihood: -7335.523952941019
GWKokab log_likelihood: -1088.054818877789
GWKokab log_likelihood: -1678.6440621163752
GWKokab log_likelihood: -1380.8451099829713
GWKokab log_likelihood: -2976.8423784504776
GWKokab log_likelihood: -867.5595684475314
GWKokab log_likelihood: -2669.273064953634
GWKokab log_likelihood: -5843.711600051809
GWKokab exp_rate_integral: 0.9165924778471404
GWKokab exp_rate_integral: 1.4594180008065099
GWKokab parameters: {'alpha_0': Array(nan, dtype=float64), 'beta_0': Array(nan, dtype=float64), 'loc_m1_0': Array(nan, dtype=float64), 'loc_m2_0': Array(nan, dtype=float64), 'log_rates': Array([nan, nan], dtype=float64), 'mean_chi1_g_0': Array(nan, dtype=float64), 'mean_chi1_pl_0': Array(nan, dtype=float64), 'mean_chi2_g_0': Array(nan, dtype=float64), 'mean_chi2_pl_0': Array(nan, dtype=float64), 'mmax_0': Array(nan, dtype=float64), 'mmin_0': Array(nan, dtype=float64), 'scale_m1_0': Array(nan, dtype=float64), 'scale_m2_0': Array(nan, dtype=float64), 'std_dev_tilt1_g_0': Array(nan, dtype=float64), 'std_dev_tilt1_pl_0': Array(nan, dtype=float64), 'std_dev_tilt2_g_0': Array(nan, dtype=float64), 'std_dev_tilt2_pl_0': Array(nan, dtype=float64), 'variance_chi1_g_0': Array(nan, dtype=float64), 'variance_chi1_pl_0': Array(nan, dtype=float64), 'variance_chi2_g_0': Array(nan, dtype=float64), 'variance_chi2_pl_0': Array(nan, dtype=float64)}
GWKokab log_prior: -inf
GWKokab log_prior: -inf
GWKokab log_prior: -inf
GWKokab log_prior: -inf
GWKokab log_prior: -inf
GWKokab log_prior: -inf
GWKokab log_prior: -inf
GWKokab log_prior: -inf
GWKokab log_prior: -inf
GWKokab log_prior: -inf
GWKokab parameters: {'alpha_0': Array(nan, dtype=float64), 'beta_0': Array(nan, dtype=float64), 'loc_m1_0': Array(nan, dtype=float64), 'loc_m2_0': Array(nan, dtype=float64), 'log_rates': Array([nan, nan], dtype=float64), 'mean_chi1_g_0': Array(nan, dtype=float64), 'mean_chi1_pl_0': Array(nan, dtype=float64), 'mean_chi2_g_0': Array(nan, dtype=float64), 'mean_chi2_pl_0': Array(nan, dtype=float64), 'mmax_0': Array(nan, dtype=float64), 'mmin_0': Array(nan, dtype=float64), 'scale_m1_0': Array(nan, dtype=float64), 'scale_m2_0': Array(nan, dtype=float64), 'std_dev_tilt1_g_0': Array(nan, dtype=float64), 'std_dev_tilt1_pl_0': Array(nan, dtype=float64), 'std_dev_tilt2_g_0': Array(nan, dtype=float64), 'std_dev_tilt2_pl_0': Array(nan, dtype=float64), 'variance_chi1_g_0': Array(nan, dtype=float64), 'variance_chi1_pl_0': Array(nan, dtype=float64), 'variance_chi2_g_0': Array(nan, dtype=float64), 'variance_chi2_pl_0': Array(nan, dtype=float64)}
GWKokab parameters: {'alpha_0': Array(nan, dtype=float64), 'beta_0': Array(nan, dtype=float64), 'loc_m1_0': Array(nan, dtype=float64), 'loc_m2_0': Array(nan, dtype=float64), 'log_rates': Array([nan, nan], dtype=float64), 'mean_chi1_g_0': Array(nan, dtype=float64), 'mean_chi1_pl_0': Array(nan, dtype=float64), 'mean_chi2_g_0': Array(nan, dtype=float64), 'mean_chi2_pl_0': Array(nan, dtype=float64), 'mmax_0': Array(nan, dtype=float64), 'mmin_0': Array(nan, dtype=float64), 'scale_m1_0': Array(nan, dtype=float64), 'scale_m2_0': Array(nan, dtype=float64), 'std_dev_tilt1_g_0': Array(nan, dtype=float64), 'std_dev_tilt1_pl_0': Array(nan, dtype=float64), 'std_dev_tilt2_g_0': Array(nan, dtype=float64), 'std_dev_tilt2_pl_0': Array(nan, dtype=float64), 'variance_chi1_g_0': Array(nan, dtype=float64), 'variance_chi1_pl_0': Array(nan, dtype=float64), 'variance_chi2_g_0': Array(nan, dtype=float64), 'variance_chi2_pl_0': Array(nan, dtype=float64)}
GWKokab parameters: {'alpha_0': Array(nan, dtype=float64), 'beta_0': Array(nan, dtype=float64), 'loc_m1_0': Array(nan, dtype=float64), 'loc_m2_0': Array(nan, dtype=float64), 'log_rates': Array([nan, nan], dtype=float64), 'mean_chi1_g_0': Array(nan, dtype=float64), 'mean_chi1_pl_0': Array(nan, dtype=float64), 'mean_chi2_g_0': Array(nan, dtype=float64), 'mean_chi2_pl_0': Array(nan, dtype=float64), 'mmax_0': Array(nan, dtype=float64), 'mmin_0': Array(nan, dtype=float64), 'scale_m1_0': Array(nan, dtype=float64), 'scale_m2_0': Array(nan, dtype=float64), 'std_dev_tilt1_g_0': Array(nan, dtype=float64), 'std_dev_tilt1_pl_0': Array(nan, dtype=float64), 'std_dev_tilt2_g_0': Array(nan, dtype=float64), 'std_dev_tilt2_pl_0': Array(nan, dtype=float64), 'variance_chi1_g_0': Array(nan, dtype=float64), 'variance_chi1_pl_0': Array(nan, dtype=float64), 'variance_chi2_g_0': Array(nan, dtype=float64), 'variance_chi2_pl_0': Array(nan, dtype=float64)}
GWKokab parameters: {'alpha_0': Array(nan, dtype=float64), 'beta_0': Array(nan, dtype=float64), 'loc_m1_0': Array(nan, dtype=float64), 'loc_m2_0': Array(nan, dtype=float64), 'log_rates': Array([nan, nan], dtype=float64), 'mean_chi1_g_0': Array(nan, dtype=float64), 'mean_chi1_pl_0': Array(nan, dtype=float64), 'mean_chi2_g_0': Array(nan, dtype=float64), 'mean_chi2_pl_0': Array(nan, dtype=float64), 'mmax_0': Array(nan, dtype=float64), 'mmin_0': Array(nan, dtype=float64), 'scale_m1_0': Array(nan, dtype=float64), 'scale_m2_0': Array(nan, dtype=float64), 'std_dev_tilt1_g_0': Array(nan, dtype=float64), 'std_dev_tilt1_pl_0': Array(nan, dtype=float64), 'std_dev_tilt2_g_0': Array(nan, dtype=float64), 'std_dev_tilt2_pl_0': Array(nan, dtype=float64), 'variance_chi1_g_0': Array(nan, dtype=float64), 'variance_chi1_pl_0': Array(nan, dtype=float64), 'variance_chi2_g_0': Array(nan, dtype=float64), 'variance_chi2_pl_0': Array(nan, dtype=float64)}
GWKokab parameters: {'alpha_0': Array(nan, dtype=float64), 'beta_0': Array(nan, dtype=float64), 'loc_m1_0': Array(nan, dtype=float64), 'loc_m2_0': Array(nan, dtype=float64), 'log_rates': Array([nan, nan], dtype=float64), 'mean_chi1_g_0': Array(nan, dtype=float64), 'mean_chi1_pl_0': Array(nan, dtype=float64), 'mean_chi2_g_0': Array(nan, dtype=float64), 'mean_chi2_pl_0': Array(nan, dtype=float64), 'mmax_0': Array(nan, dtype=float64), 'mmin_0': Array(nan, dtype=float64), 'scale_m1_0': Array(nan, dtype=float64), 'scale_m2_0': Array(nan, dtype=float64), 'std_dev_tilt1_g_0': Array(nan, dtype=float64), 'std_dev_tilt1_pl_0': Array(nan, dtype=float64), 'std_dev_tilt2_g_0': Array(nan, dtype=float64), 'std_dev_tilt2_pl_0': Array(nan, dtype=float64), 'variance_chi1_g_0': Array(nan, dtype=float64), 'variance_chi1_pl_0': Array(nan, dtype=float64), 'variance_chi2_g_0': Array(nan, dtype=float64), 'variance_chi2_pl_0': Array(nan, dtype=float64)}
GWKokab parameters: {'alpha_0': Array(nan, dtype=float64), 'beta_0': Array(nan, dtype=float64), 'loc_m1_0': Array(nan, dtype=float64), 'loc_m2_0': Array(nan, dtype=float64), 'log_rates': Array([nan, nan], dtype=float64), 'mean_chi1_g_0': Array(nan, dtype=float64), 'mean_chi1_pl_0': Array(nan, dtype=float64), 'mean_chi2_g_0': Array(nan, dtype=float64), 'mean_chi2_pl_0': Array(nan, dtype=float64), 'mmax_0': Array(nan, dtype=float64), 'mmin_0': Array(nan, dtype=float64), 'scale_m1_0': Array(nan, dtype=float64), 'scale_m2_0': Array(nan, dtype=float64), 'std_dev_tilt1_g_0': Array(nan, dtype=float64), 'std_dev_tilt1_pl_0': Array(nan, dtype=float64), 'std_dev_tilt2_g_0': Array(nan, dtype=float64), 'std_dev_tilt2_pl_0': Array(nan, dtype=float64), 'variance_chi1_g_0': Array(nan, dtype=float64), 'variance_chi1_pl_0': Array(nan, dtype=float64), 'variance_chi2_g_0': Array(nan, dtype=float64), 'variance_chi2_pl_0': Array(nan, dtype=float64)}
GWKokab parameters: {'alpha_0': Array(nan, dtype=float64), 'beta_0': Array(nan, dtype=float64), 'loc_m1_0': Array(nan, dtype=float64), 'loc_m2_0': Array(nan, dtype=float64), 'log_rates': Array([nan, nan], dtype=float64), 'mean_chi1_g_0': Array(nan, dtype=float64), 'mean_chi1_pl_0': Array(nan, dtype=float64), 'mean_chi2_g_0': Array(nan, dtype=float64), 'mean_chi2_pl_0': Array(nan, dtype=float64), 'mmax_0': Array(nan, dtype=float64), 'mmin_0': Array(nan, dtype=float64), 'scale_m1_0': Array(nan, dtype=float64), 'scale_m2_0': Array(nan, dtype=float64), 'std_dev_tilt1_g_0': Array(nan, dtype=float64), 'std_dev_tilt1_pl_0': Array(nan, dtype=float64), 'std_dev_tilt2_g_0': Array(nan, dtype=float64), 'std_dev_tilt2_pl_0': Array(nan, dtype=float64), 'variance_chi1_g_0': Array(nan, dtype=float64), 'variance_chi1_pl_0': Array(nan, dtype=float64), 'variance_chi2_g_0': Array(nan, dtype=float64), 'variance_chi2_pl_0': Array(nan, dtype=float64)}
GWKokab parameters: {'alpha_0': Array(nan, dtype=float64), 'beta_0': Array(nan, dtype=float64), 'loc_m1_0': Array(nan, dtype=float64), 'loc_m2_0': Array(nan, dtype=float64), 'log_rates': Array([nan, nan], dtype=float64), 'mean_chi1_g_0': Array(nan, dtype=float64), 'mean_chi1_pl_0': Array(nan, dtype=float64), 'mean_chi2_g_0': Array(nan, dtype=float64), 'mean_chi2_pl_0': Array(nan, dtype=float64), 'mmax_0': Array(nan, dtype=float64), 'mmin_0': Array(nan, dtype=float64), 'scale_m1_0': Array(nan, dtype=float64), 'scale_m2_0': Array(nan, dtype=float64), 'std_dev_tilt1_g_0': Array(nan, dtype=float64), 'std_dev_tilt1_pl_0': Array(nan, dtype=float64), 'std_dev_tilt2_g_0': Array(nan, dtype=float64), 'std_dev_tilt2_pl_0': Array(nan, dtype=float64), 'variance_chi1_g_0': Array(nan, dtype=float64), 'variance_chi1_pl_0': Array(nan, dtype=float64), 'variance_chi2_g_0': Array(nan, dtype=float64), 'variance_chi2_pl_0': Array(nan, dtype=float64)}
GWKokab parameters: {'alpha_0': Array(nan, dtype=float64), 'beta_0': Array(nan, dtype=float64), 'loc_m1_0': Array(nan, dtype=float64), 'loc_m2_0': Array(nan, dtype=float64), 'log_rates': Array([nan, nan], dtype=float64), 'mean_chi1_g_0': Array(nan, dtype=float64), 'mean_chi1_pl_0': Array(nan, dtype=float64), 'mean_chi2_g_0': Array(nan, dtype=float64), 'mean_chi2_pl_0': Array(nan, dtype=float64), 'mmax_0': Array(nan, dtype=float64), 'mmin_0': Array(nan, dtype=float64), 'scale_m1_0': Array(nan, dtype=float64), 'scale_m2_0': Array(nan, dtype=float64), 'std_dev_tilt1_g_0': Array(nan, dtype=float64), 'std_dev_tilt1_pl_0': Array(nan, dtype=float64), 'std_dev_tilt2_g_0': Array(nan, dtype=float64), 'std_dev_tilt2_pl_0': Array(nan, dtype=float64), 'variance_chi1_g_0': Array(nan, dtype=float64), 'variance_chi1_pl_0': Array(nan, dtype=float64), 'variance_chi2_g_0': Array(nan, dtype=float64), 'variance_chi2_pl_0': Array(nan, dtype=float64)}
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
GWKokab model_log_likelihood: nan
GWKokab model_log_likelihood: nan
GWKokab model_log_likelihood: nan
GWKokab model_log_likelihood: nan
GWKokab model_log_likelihood: nan
GWKokab model_log_likelihood: nan
GWKokab model_log_likelihood: nan
GWKokab model_log_likelihood: nan
GWKokab model_log_likelihood: nan
GWKokab model_log_likelihood: nan
GWKokab log_likelihood: nan
GWKokab log_likelihood: nan
GWKokab log_likelihood: nan
GWKokab log_likelihood: nan
GWKokab log_likelihood: nan
GWKokab log_likelihood: nan
GWKokab log_likelihood: nan
GWKokab log_likelihood: nan
GWKokab log_likelihood: nan
GWKokab log_likelihood: nan
GWKokab exp_rate_integral: nan
GWKokab exp_rate_integral: nan
Traceback (most recent call last):
  File "/home/muhammad.zeeshan/.conda/envs/kvenv/bin/sage_n_pls_m_gs", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/muhammad.zeeshan/.conda/envs/kvenv/lib/python3.11/site-packages/kokab/n_pls_m_gs/sage.py", line 192, in main
    handler.run()
  File "/home/muhammad.zeeshan/.conda/envs/kvenv/lib/python3.11/site-packages/gwkokab/_src/inference/flowMChandler.py", line 90, in run
    sampler.sample(self.initial_position, self.data)
  File "/home/muhammad.zeeshan/.conda/envs/kvenv/lib/python3.11/site-packages/flowMC/Sampler.py", line 204, in sample
    ) = strategy(
        ^^^^^^^^^
  File "/home/muhammad.zeeshan/.conda/envs/kvenv/lib/python3.11/site-packages/flowMC/strategy/global_tuning.py", line 100, in __call__
    summary["chains"] = jnp.append(
                        ^^^^^^^^^^^
  File "/home/muhammad.zeeshan/.conda/envs/kvenv/lib/python3.11/site-packages/jax/_src/numpy/lax_numpy.py", line 5464, in append
    return concatenate([arr, values], axis=axis)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/muhammad.zeeshan/.conda/envs/kvenv/lib/python3.11/site-packages/jax/_src/numpy/lax_numpy.py", line 3151, in concatenate
    arrays_out = [lax.concatenate(arrays_out[i:i+k], axis)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/muhammad.zeeshan/.conda/envs/kvenv/lib/python3.11/site-packages/jax/_src/numpy/lax_numpy.py", line 3151, in <listcomp>
    arrays_out = [lax.concatenate(arrays_out[i:i+k], axis)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Cannot concatenate arrays with shapes that differ in dimensions other than the one being concatenated: concatenating along dimension 1 for shapes (10, 0, 10), (10, 100, 22).
--------------------
For simplicity, JAX has removed its internal frames from the traceback of the following exception. Set JAX_TRACEBACK_FILTERING=off to include these.`

**Comment by Qazalbash** [comment]
@coderabbitai full review

File: src/kokab/utils/common.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    :return: dictionary of posterior data
+    """
+    data_set = {
+        "data": [
+            pd.read_csv(event, delimiter=" ")[posterior_columns].to_numpy()
+            for event in filenames
+        ],
+        "N": len(filenames),
+    }
+    return data_set
```
[the remark this answers is not in the document]
Comment (Qazalbash) [reply src/kokab/utils/common.py:80]: Can you suggest the code?

File: src/kokab/utils/common.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    .. code:: python
+
+        >>> expand_arguments("physics", 3)
+        ["physics_0", "physics_1", "physics_2"]
+
+    :param arg: argument to extend
+    :param n: number of strings to extend
+    :return: list of extended arguments
+    """
+    return list(map(lambda i: arg + f"_{i}", range(n)))
```
[the remark this answers is not in the document]
Comment (Qazalbash) [reply src/kokab/utils/common.py:39]: Can you make this change?

[the remark this answers is not in the document]
Comment (Qazalbash) [reply src/kokab/utils/common.py]: it is fixed in 8dafcb4.

File: src/kokab/utils/common.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    flowMC_json["nf_model_kwargs"]["model"] = "MaskedCouplingRQSpline"
+
+    flowMC_json["sampler_kwargs"]["data"] = None
+    flowMC_json["sampler_kwargs"]["logging"] = True
+    flowMC_json["sampler_kwargs"]["outdir"] = "inf-plot"
+    flowMC_json["sampler_kwargs"]["precompile"] = False
+    flowMC_json["sampler_kwargs"]["use_global"] = True
+    flowMC_json["sampler_kwargs"]["verbose"] = False
+
+    return flowMC_json
```
[the remark this answers is not in the document]
Comment (Qazalbash) [reply src/kokab/utils/common.py]: it is fixed in 8dafcb4b7772d9e86bd5f6f012e6f3aac8c5ecb0.

File: src/kokab/utils/common.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Qazalbash) [reply src/kokab/utils/common.py]: It is fixed in f38e383.

**Comment by Qazalbash** [comment]
@coderabbitai full review

**Comment by Qazalbash** [comment]
@coderabbitai full review

**Pull request merged**